components/sections/OurWork.jsx

```diff
@@ -0,0 +1,23 @@
+import Title from "../ui/Title";
+import work from "../../data/ourWork";
+import Work from "../cards/Work";
+
+const OurWork = () => {
+  return (
+    <div className="flex flex-col ">
+      <Title content="Our Work" styles={'ml-20'}/>
+      <div>
```

Comment:
![image](https://user-images.githubusercontent.com/90575886/211782247-e95b7989-8094-45a3-95ce-257b9045fb27.png)
 There is space on both sides whereas here you only have space on the left